test/parallel/test-repl-preview-newlines.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const common = require('../common');
+const ArrayStream = require('../common/arraystream');
+const assert = require('assert');
+const repl = require('repl');
+
+common.skipIfInspectorDisabled();
+
+const inputStream = new ArrayStream();
+const outputStream = new ArrayStream();
+repl.start({
+  input: inputStream,
+  output: outputStream,
+  useGlobal: false,
+  terminal: true,
+  useColors: true
+});
+
+let output = '';
+outputStream.write = (chunk) => output += chunk;
+
+const testChars = ['\\n', '\\v', '\\r'];
+
+for (const test of testChars) {
+  inputStream.emit('data', `"${test}"()`);
+  // Make sure the output is on a single line
+  assert.strictEqual(output, `"${test}"()\n\x1B[90mTypeError: "\x1B[39m\x1B[9G\x1B[1A`);
```

Comment:
```suggestion
for (const testChar of '\n\v\r') {
  inputStream.emit('data', `${JSON.stringify(testChar)}()`);
  // Make sure the output is on a single line
  assert.strictEqual(output, `${JSON.stringify(testChar)}()\n\x1B[90mTypeError: "\x1B[39m\x1B[9G\x1B[1A`);
```

Reply:
@aduh95 It's not `\n`, it's `\\n`, which is two characters.

Reply:
`JSON.stringify('\n') === "\\n"`

Reply:
Oh I didn't see that part... thank you!